schema.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals(
+    id INTEGER,
```

Comment:
- [ ] [OPTIONAL] You could make the id column not null and autoincremented `id int NOT NULL IDENTITY(1,1)`
- [ ] [OPTIONAL] You could make the id a primary key `PRIMARY KEY (id)`